### PR TITLE
release 1.1.0 deployments

### PR DESCRIPTION
**Pull request by pohly**
**What type of PR is this?**
/kind api-change

**What this PR does / why we need it**:

This is needed because we want the v1.1.0 release of the driver to have the corresponding deployments that use that driver.

**Special notes for your reviewer**:

The first commit could already be merged now. Let me know if you want to do that and I'll create a separate PR for it. The second commit should be merged only shortly before the actual v1.1.0 release, because merging it will break the deployments and thus the periodic CI testing until the release image becomes available.

**Does this PR introduce a user-facing change?**:

```release-note
deployments for Kubernetes 1.13 and 1.14 use the v1.1.0 driver image
```

### CODE REVIEW

File: deploy/kubernetes-1.14/hostpath/csi-hostpath-plugin.yaml [hunk 1]
```diff
@@ -63,7 +63,7 @@ spec:
             name: csi-data-dir
 
         - name: hostpath
-          image: quay.io/k8scsi/hostpathplugin:canary # TODO: replace with v1.1.0 when it is cut
```
Comment (msau42) [inline deploy/kubernetes-1.14/hostpath/csi-hostpath-plugin.yaml]: do we still want a deployment for master?

Comment (pohly) [reply deploy/kubernetes-1.14/hostpath/csi-hostpath-plugin.yaml]: No. The CI only uses "kubernetes-x.yy". Using "master" as directory and "kubernetes-1.14" as symlink was an interim solution to indicate to users that the deployment hadn't been finalized yet. That's no longer the case.

I also can't think of another reason right why we might want to have a separate deployment for Kubernetes master. If someone wants to try out the canary images, then they can do that with `IMAGE_TAG=canary deploy/kubernetes-latest/deploy-hostpath.sh`.

**Comment by msau42** [comment]
Oh one more thing. Can you help prepare a changelog file similar to the sidecars that summarize the important features and fixes for this release?

**Comment by pohly** [comment]
Michelle Au <notifications@github.com> writes:
> Oh one more thing. Can you help prepare a changelog file similar to
> the sidecars that summarize the important features and fixes for this
> release?

I had the same thought and actually tried, but the auto-generated
changelog from the release-notes tools is pretty useless for this repo
(just a single entry because most of the PRs don't have release
notes). I then had to stop due to lack of time.

Looking at the commit log I'm not sure which changes need to be
mentioned.

Given that this is a demo driver, perhaps we can simply release 1.1.0
without a changelog and then become more diligent about it for future
releases?

**Comment by msau42** [comment]
That's fine. What were the main changes we made? Added raw block support, improved deployment example, added snapshot examples

**Pull request merged**